services/audit-service/src/test/java/com/sourcefuse/jarc/services/auditservice/AuditControllerTests.java

```diff
@@ -0,0 +1,359 @@
+package com.sourcefuse.jarc.services.auditservice;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.sourcefuse.jarc.core.adapters.LocalDateTimeTypeAdapter;
+import com.sourcefuse.jarc.core.enums.AuditActions;
+import com.sourcefuse.jarc.services.auditservice.models.AuditLog;
+import com.sourcefuse.jarc.services.auditservice.repositories.AuditLogRepository;
+import com.sourcefuse.jarc.services.auditservice.test.models.Role;
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.PersistenceContext;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class AuditControllerTests {
+
+  @PersistenceContext
+  private EntityManager entityManager;
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @Autowired
+  private AuditLogRepository auditLogRepository;
+
+  ObjectMapper objectMapper = new ObjectMapper();
+
+  UUID mockUserId = UUID.fromString("db66f86d-a7e8-45b7-a6ad-8a5024158377");
+  UUID mockTempRoleId = UUID.fromString("c87a2848-ce70-41a9-95c7-d7ab230ebcac");
+  UUID mockUserRoleId = UUID.fromString("0836f6a1-a75e-4468-999a-fa7e61492c98");
+
+  AuditLog auditLog1, auditLog2, auditLog;
+
+  Gson gson = new GsonBuilder()
+    .registerTypeAdapter(LocalDateTime.class, new LocalDateTimeTypeAdapter())
+    .create();
+
+  @BeforeEach
+  void clearUserAndAuditLog() {
+    this.clearTables();
+    this.addDefaultLogs();
+  }
+
+  @Test
+  void testCreateAuditLog() throws Exception {
+    this.mockMvc.perform(
+        post("/audit_logs")
+          .contentType("application/json")
```

Comment:
Can we create an enum for content type in core to be consumed across the packages